### ISSUE TITLE
Documentation: "spring.ai.vectorstore.redis.uri" property is not available anymore

### ISSUE THREAD

**Issue by timosalm** [spring-projects/spring-ai#1789]
**Bug description**
The `spring.ai.vectorstore.redis.uri` property still mentioned in the docs [here](https://docs.spring.io/spring-ai/reference/1.0/api/vectordbs/redis.html) was removed in Spring AI M3.

`spring.data.redis` properties must be used to configure the datasource now.

**Environment**
Spring AI M3/M4

**Steps to reproduce**
-

**Expected behavior**
Documentation should be updated.

**Comment by dashaun** [comment]
@timosalm When you populate the ```spring.data.redis``` properties, with ```spring-ai-redis-store-spring-boot-starter``` do you get an auto-configured VectorStore?

I can't seem to configure a ```RedisVectorStore``` at all with Spring AI M4 at the moment, auto-configured or manual.

**Comment by dashaun** [comment]
I had to remove ```spring-boot-starter-data-redis``` to get a ```RedisVectorStore```

**Comment by timosalm** [comment]
@dashaun Yes, I don't have a "spring-boot-starter-data-redis" dependency: https://github.com/timosalm/spring-ai/blob/main/build.gradle

Maybe you should create an issue for that, as there will probably be applications that use Redis not only as a vector store, and at least some guidance in the docs would be great.
